src/actions/chatActions.js

```diff
@@ -143,6 +149,49 @@ export const sendMessageByContactAction = (username: string, message: Object) =>
   };
 };
 
+export const getChatDraftByContactAction = (contactId: string) => {
+  return async (dispatch: Function) => {
+    try {
+      const { drafts } = await storage.get('chat');
```

Comment:
I think if we put here `const { drafts = [] }` then we could avoid wrapping everything into try..catch.
What do you think?

Reply:
The reason for the try/catch is mostly for the `storage.get('chat')` part, if for some reason Storage crashes.

Setting a default value should be there, so thanks for noticing it.

Reply:
If for some reason Storage crashes we should probably block the app and notify the user...
If you think `storage.get` could crash, then it's better to wrap the code of this `.get()` method inside the storage.js file as we're using `storage.get()` in a lot of places across the app

Reply:
Btw Sentry wrapps the whole codebase with the try..catch by default

Reply:
ok 👍